Procfile.dev

```diff
@@ -0,0 +1,4 @@
+web: rails s
+hot: sh -c 'cd client && npm start'
```

Comment:
`hot` runs the command to run your client side app using the [webpack dev server](https://webpack.github.io/docs/webpack-dev-server.html). This allows you to edit the JS and SCSS files and see your UI do "Hot Module Reloading" which is like "Live Reload" except any browser state is typically maintained.

Note, this is ONLY used for development testing.

See file [server.js](https://github.com/shakacode/react_on_rails-generator-results/pull/123/files#diff-87c30590767fb019a17f8fc0cef64ddc) for how we set that up.

While there are techniques to proxy the API request to your rails server on port 3000, we don't do that. You have to create simple mocks.